database/updateCmd.go

```diff
@@ -0,0 +1,24 @@
+package db
```

Comment:
Please move this to the `cmds` package.

Reply:
I’d like to only reference the urfave cli package in the cmds package, and allow the rest of the code to run on its own.

Hope that makes sense

Reply:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**🎯 suggestion:** ‏I too would suggest that the command part be moved to the `cmd` folder as that would be the command part. 
Then what we can do as part of initialising that command could be something like :
```go
type Storage interface {
	CreateEntry(*Entry) error
	UpdateEntry(string, string, interface{}) error
	DeleteEntry(string) error
	GetAllEntries() ([]Entry, error)
	GetByName(string) (Entry, error)
}

NewStorageCommand(storage Storage) *cli.Command
```
This way we could move the actual storage implementation to something like `internal/storage/sqlite_storage.go` and this way when we initiate the command we could swap out the implementation in the future based on config file or flag like say `sesh connect . -db=redis` and the code would initiate the redis implementation of the DB interface and pass it in.

This means the command and the DB implementation have proper separation of concerns.

Reply:
ive refactored it to have the commands living in the cmds package